contrib/kind.sh

```diff
@@ -205,307 +203,339 @@ print_params()
 
-  # Build ovn kube image
+  # Create ovn-kube manifests
   pushd ../dist/images
-  # Find all built executables, but ignore the 'windows' directory if it exists
-  BINS=$(find ../../go-controller/_output/go/bin/ -maxdepth 1 -type f | xargs)
-  sudo cp -f ${BINS} .
-  echo "ref: $(git rev-parse  --symbolic-full-name HEAD)  commit: $(git rev-parse  HEAD)" > git_info
-  docker build -t ovn-daemonset-f:dev -f Dockerfile.fedora .
-  OVN_IMAGE=ovn-daemonset-f:dev
+  ./daemonset.sh \
+    --image=${OVN_IMAGE} \
+    --net-cidr=${NET_CIDR} \
+    --svc-cidr=${SVC_CIDR} \
+    --gateway-mode=${OVN_GATEWAY_MODE} \
+    --hybrid-enabled=${OVN_HYBRID_OVERLAY_ENABLE} \
+    --disable-snat-multiple-gws=${OVN_DISABLE_SNAT_MULTIPLE_GWS} \
+    --ovn-empty-lb-events=${OVN_EMPTY_LB_EVENTS} \
+    --multicast-enabled=${OVN_MULTICAST_ENABLE} \
+    --k8s-apiserver=${API_URL} \
+    --ovn-master-count=${KIND_NUM_MASTER} \
+    --kind \
+    --ovn-unprivileged-mode=no \
+    --master-loglevel=${MASTER_LOG_LEVEL} \
+    --node-loglevel=${NODE_LOG_LEVEL} \
+    --dbchecker-loglevel=${DBCHECKER_LOG_LEVEL}\
+    --ovn-loglevel-northd="${OVN_LOG_LEVEL_NORTHD}" \
+    --ovn-loglevel-nb="${OVN_LOG_LEVEL_NB}" \
+    --ovn-loglevel-sb="${OVN_LOG_LEVEL_SB}" \
+    --ovn-loglevel-controller="${OVN_LOG_LEVEL_CONTROLLER}" \
+    --ovn-loglevel-nbctld="${OVN_LOG_LEVEL_NBCTLD}" \
+    --egress-ip-enable=true\
+    --v4-join-subnet=${JOIN_SUBNET_IPV4}\
+    --v6-join-subnet=${JOIN_SUBNET_IPV6}
   popd
-fi
-
-# Detect API IP address for OVN
-
-# Despite OVN run in pod they will only obtain the VIRTUAL apiserver address
-# and since OVN has to provide the connectivity to service
-# it can not be bootstrapped
-
-# This is the address of the node with the control-plane
-API_URL=$(kind get kubeconfig --internal --name ${KIND_CLUSTER_NAME} | grep server | awk '{ print $2 }')
 
-# Create ovn-kube manifests
-pushd ../dist/images
-./daemonset.sh \
-  --image=${OVN_IMAGE} \
-  --net-cidr=${NET_CIDR} \
-  --svc-cidr=${SVC_CIDR} \
-  --gateway-mode=${OVN_GATEWAY_MODE} \
-  --hybrid-enabled=${OVN_HYBRID_OVERLAY_ENABLE} \
-  --disable-snat-multiple-gws=${OVN_DISABLE_SNAT_MULTIPLE_GWS} \
-  --ovn-empty-lb-events=${OVN_EMPTY_LB_EVENTS} \
-  --multicast-enabled=${OVN_MULTICAST_ENABLE} \
-  --k8s-apiserver=${API_URL} \
-  --ovn-master-count=${KIND_NUM_MASTER} \
-  --kind \
-  --ovn-unprivileged-mode=no \
-  --master-loglevel=${MASTER_LOG_LEVEL} \
-  --node-loglevel=${NODE_LOG_LEVEL} \
-  --dbchecker-loglevel=${DBCHECKER_LOG_LEVEL}\
-  --ovn-loglevel-northd="${OVN_LOG_LEVEL_NORTHD}" \
-  --ovn-loglevel-nb="${OVN_LOG_LEVEL_NB}" \
-  --ovn-loglevel-sb="${OVN_LOG_LEVEL_SB}" \
-  --ovn-loglevel-controller="${OVN_LOG_LEVEL_CONTROLLER}" \
-  --ovn-loglevel-nbctld="${OVN_LOG_LEVEL_NBCTLD}" \
-  --egress-ip-enable=true\
-  --v4-join-subnet=${JOIN_SUBNET_IPV4}\
-  --v6-join-subnet=${JOIN_SUBNET_IPV6}
-popd
+  kind load docker-image ${OVN_IMAGE} --name ${KIND_CLUSTER_NAME}
```

Comment:
this doesn't match the function description `create_ovn_kube_manifests`, this has to be after `build_ovn_image` or change the function adding this  to `install_ovn_image`

Reply:
moved this line into `install_ovn_image`